.github/workflows/ci-on-main.yml

```
@@ -69,3 +60,29 @@ jobs:
             -destination 'generic/platform=iOS Simulator' \
             -derivedDataPath build \
             clean build
+
+  automerge:
+    name: 🚀 Auto-merge develop → main (keep develop)
+    needs: [lint-test-build-ios]
+    runs-on: ubuntu-latest
+    if: >
```

actionlint (1.7.7)

if: condition "${{\n  github.event_name == 'pull_request' &&\n  github.event.pull_request.base.ref == 'main' &&\n  github.event.pull_request.head.ref == 'develop' &&\n  github.event.pull_request.state != 'closed' }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)